draft-ietf-protected-audience-key-value-server-api.md

```diff
@@ -186,34 +317,49 @@ extended diagnostic notation from {{CDDL}} appendix G:
     }
   ]
 }
-~~~
+~~~~~
 
-### Schema of the Response
+## Core Response Data {#response}
 
-#### Response
+The response is compressed. The compression is applied independently to each compression group. That
+means, the response object mainly contains a list of compressed blobs, each for one compression
+group. Each blob is for outputs of one or more partitions, sharing the same `compressionGroup` value
+as specified in the request.
 
-The response is compressed. Due to security and privacy reasons the compression is applied independently to each compression group. That means, The response object mainly contains a list of compressed blobs, each for one compression group. Each blob is for outputs of one or more partitions, sharing the same compressionGroup value as specified in the request.
+### Encryption {#response-encryption}
 
-~~~ cddl
+The response uses the a similar encapsulated response format to that used by [OHTTP] (see
+{{Section 4.4 of OHTTP}}), but with the custom `message/ad-auction-trusted-signals-request`
```

Comment:
Should the response have the response media type? not request?

Reply:
Done.